src/ai/backend/manager/api/resource.py

```diff
@@ -332,8 +332,12 @@ def _pipe_builder(r: Redis) -> RedisPipeline:
             continue
         last_stat = msgpack.unpackb(raw_stat)
         nfs = None
-        if row['mounts'] is not None:
-            nfs = list(set([mount[1] for mount in row['mounts']]))
+        if row['vfolder_mounts']:
+            # For >=22.03, return used host directories instead of volume host, which is not so useful.
+            nfs = list(set([str(mount.host_path) for mount in row['vfolder_mounts']]))
+        elif row['mounts'] and len(row['mounts']) > 0 and isinstance(row['mounts'][0], list):
```

Comment:
```suggestion
        elif row['mounts'] and isinstance(row['mounts'][0], list):
```
You may rely on the Python's truthness semantic about the emptiness of lists as PEP-8 recommends.